ONSdigital/florence
# Disable Florence during request

### What

Disable Florence's UI during any update to content in attempt to fix data loss issues on page save.

The screen will be blanked out and show a loading icon whilst a request is in progress:
![localhost-8081-florence-index html pst screen](https://user-images.githubusercontent.com/8183179/26928520-512d25ba-4c4e-11e7-912e-d4c9c95b0f3c.png)

The aim of this is to prevent any user interaction happening whilst a request is being made.

### How to review

- Checkout this branch
- Start up Florence
- Edit any item of content
- Check that clicking is disabled during any request sent from client to Zebedee to update the content.
- Check there are no obvious adverse effects on other screens.

### Who can review

Anyone but me.
